libs/pages/logs/environment/src/lib/ui/row-pod/row-pod.tsx

```diff
@@ -102,11 +74,15 @@ export function RowPod(props: RowPodProps) {
           )}
         </div>
         <div data-testid="cell-date" className="px-4 pt-0.5 text-neutral-350 whitespace-nowrap">
-          {dateFullFormat(data.created_at, utc ? 'UTC' : undefined, 'dd MMM, HH:mm:ss:SS')}
-        </div>
-        <div data-testid="cell-msg" className="select-text pr-6 pt-0.5 text-neutral-50 relative w-full">
-          <span className="whitespace-pre-wrap break-all">{convertToAnsi(data.message)}</span>
+          {dateFullFormat(data.created_at, utc ? 'UTC' : timeZone, 'dd MMM, HH:mm:ss:SS')}
         </div>
+        <Ansi
+          data-testid="cell-msg"
+          className="select-text pr-6 pt-0.5 text-neutral-50 relative w-full whitespace-pre-wrap break-all code-ansi"
+          linkify
```

Comment:
Why you didn't use convertToAnsi that uniformize the usage of the Ansi package ?

Reply:
https://github.com/Qovery/console/pull/811/commits/c465c335df0b24a1bb94c415984287bd19ff0502